Makefile.dep

```diff
@@ -396,8 +396,20 @@ ifneq (,$(filter stdio_rtt,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
-ifneq (,$(filter stdio_uart,$(USEMODULE)))
+ifneq (,$(filter shell,$(USEMODULE)))
+  USEMODULE += stdio_uart_rx
```

Comment:
this'll not work with stdio_rtt

Reply:
maybe depend on an intermediate module like "stdio_input" and make that depend on stdio_uart_rx if stdio_uart is used?

Reply:
just checked for `stdio_uart` being used, is it good enough?